pysynchronizer/utils.py

```diff
@@ -0,0 +1,38 @@
+
+import os
+
+from uuid import UUID
+
+def is_valid_uuid(uuid_to_test):
+    try:
+        uuid_obj = UUID(uuid_to_test)
+    except ValueError:
+        return False
+    return str(uuid_obj) == uuid_to_test
+
+def generate_uuid():
+    return str(uuid.uuid4())
```

Comment:
I don't see ```uuid``` in scope. Maybe that ```from uuid import UUID``` was once just ```import uuid```? Not sure.

Reply:
Good question. I can just tack on `uuid4` to the `from uuid`.